lib/protoparser/opentelemetry/stream/streamparser.go

```diff
@@ -13,11 +14,17 @@ import (
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/fasttime"
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/prompbmarshal"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/promrelabel"
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/protoparser/common"
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/protoparser/opentelemetry/pb"
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/writeconcurrencylimiter"
 )
 
+var (
+	// sanitizeMetricName controls sanitizing metric names ingested via OpenTelemetry protocol.
+	sanitizeMetricName = flag.Bool("opentelemetry.sanitizeMetricName", false, "Sanitize metric names for the ingested OpenTelemetry data")
```

Comment:
It'd great  to sanitize all label names as well. 

Without it, it ll be not possible to use such metrics with grafana prometheus datasource.

Reply:
done